pandas/tests/test_frame.py

```diff
@@ -149,6 +149,12 @@ def test_get(self):
             result = df.get(None)
             self.assertIsNone(result)
 
+        # GH 6703
```

Comment:
this should be in ``tests/test_generic.py``. In fact ideally we would move the tests from where they are now (test_series/frame) and move them here (in the appopriate test classes).